app/components/swiper.js

```diff
@@ -71,7 +71,9 @@ export default class Swiper extends PureComponent {
         if (this.runOnLayout) {
             if (Platform.OS === 'ios') {
                 setTimeout(() => {
-                    this.scrollView.scrollTo({x: this.props.width * this.state.index, animated: false});
+                    if (this.scrollView) {
```

Comment:
You can move this line to be alongside the thid.runOnLayout and that will cover the one for android as well